src/main/java/com/cebanova/gitworkflow/GitWorkflow.java

```diff
@@ -0,0 +1,46 @@
+package com.cebanova.gitworkflow;
+
+public class GitWorkflow {
+    public static void main(String[] args) {
+        // Declaring a list of String variables
+        String step1 = "Perform git stash - to store the uncommitted changes";
+        String step2 = "Perform git pull - to fetch and download content from a remote repository and immediately " +
```

Comment:
Please add the step 2 with git fetch. If I am not mistaken, it was there before your latest changes.

Reply:
The git fetch command moved to step 4, because git pull was added in step 2, followed by git checkout in step 3